src/ResultWriter/WaveFieldWriter.cpp

```diff
@@ -295,18 +295,22 @@ void seissol::writer::WaveFieldWriter::init(unsigned int numVars,
 	if (pstrain || integrals) {
 		logInfo(rank) << "Initialize low order output";
 
-		// Refinement strategy (no refinement)
-		refinement::IdentityRefiner<double> lowTetRefiner;
+		// Setup the tetrahedron refinement strategy
+		refinement::TetrahedronRefiner<double>* lowTetRefiner = createRefiner(refinement);
```

Comment:
This refinement needs to be different from the regular wave field output. Otherwise the split in "low output" and "regular output" is pointless.

Reply:
yes I know, see first post of this issue:
>(at this point, the refinement level is the one from the wavefield output, but we could imaging a refinement_plasticity parameter in the parameter file).

With this draft, I just wanted to get some help making it work, and then I can refine the code with a further refinement parameter.